polaris-react/src/components/DataTable/DataTable.scss

```diff
@@ -385,13 +385,6 @@
     }
   }
 
-  .StickyTableColumnHeader-isScrolling {
-    // stylelint-disable -- generated by polaris-migrator DO NOT COPY
-    box-shadow: 1px 1px 0 0 var(--p-border-divider),
-      1px 0 1px 1px rgba(63, 63, 68, 0.05), 1px 0 3px 0 rgba(63, 63, 68, 0.15);
-    // stylelint-enable
-  }
-
```

Comment:
I couldn't find this used anywhere in the `DataTable` component. I think it was accidentally copied over from the `IndexTable`